pubsub/cloud-client/subscriber_test.py

```diff
@@ -160,6 +192,27 @@ def test_receive(publisher_client, topic, subscription, capsys):
     assert 'Message 1' in out
 
 
+def test_receive_synchronously(
+        publisher_client, topic, subscription_sync1, capsys):
+    _publish_messages(publisher_client, topic)
+
+    subscriber.receive_messages_synchronously(PROJECT, SUBSCRIPTION_SYNC1)
```

Comment:
For consistency with the `synchronous_pull_with_lease_management`, can we rename this function to `synchronous_pull`?